### PR TITLE
Add crypto payments

### PR DESCRIPTION
**Pull request by akegaviar**

### CODE REVIEW

File: docs/platform/top-up-your-balance-with-crypto.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+If your platform balance is not sufficient, your credit card will be charged the outstanding amount.
+
+You can top up your platform balance with your Coinbase account, BTC, ETH, or USDC.
+
+To top up the balance with your Coinbase account:
+
+1. Go to [Billing](https://console.chainstack.com/user/settings/billing).
+1. Click **Top up with crypto**.
+1. Select a predefined amount or type in your amount. Click **Next**.
+1. Click **Pay with Coinbase**.
```
Comment (NikitosnikN) [inline docs/platform/top-up-your-balance-with-crypto.md:22]: There is no 4th step, no **Pay with Coinbase** button

File: docs/platform/top-up-your-balance-with-crypto.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+1. Go to [Billing](https://console.chainstack.com/user/settings/billing).
+1. Click **Top up with crypto**.
+1. Select a predefined amount or type in your amount. Click **Next**.
+1. Select BTC, ETH, or USDC.
+1. Send BTC, ETH, or USDC to the respective generated address.
+
+For BTC, the amount in USD will be credited to your balance after 3 network confirmations.
+
+For ETH or USDC, the amount in USD will be credited to your balance after 35 network confirmations.
```
Comment (NikitosnikN) [inline docs/platform/top-up-your-balance-with-crypto.md:48]: `35 network confirmations.`

where did you find this info ?

Comment (akegaviar) [reply docs/platform/top-up-your-balance-with-crypto.md:48]: https://help.coinbase.com/en/coinbase/getting-started/crypto-education/btc
https://help.coinbase.com/en/coinbase/getting-started/crypto-education/usd-coin-usdc-faq
https://help.coinbase.com/en/coinbase/getting-started/crypto-education/eth

**Review by NikitosnikN — APPROVED** [review]

File: docs/platform/top-up-your-balance-with-crypto.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+meta:
+  - name: description
+    content: Top up your Chainstack balance with your Coinbase balance, BTC, ETH, or USDC.
+  - name: keywords
+    content: btc eth usdc crypto coinbase
+---
+
+# Top up your balance with crypto
+
+Chainstack deducts your membership plan and node usage costs from your platform balance.
```
Comment (easeev) [inline docs/platform/top-up-your-balance-with-crypto.md:11]: "membership plan"?

File: docs/platform/top-up-your-balance-with-crypto.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    content: Top up your Chainstack balance with your Coinbase balance, BTC, ETH, or USDC.
+  - name: keywords
+    content: btc eth usdc crypto coinbase
+---
+
+# Top up your balance with crypto
+
+Chainstack deducts your subscription and node usage costs from your account balance.
+
+If your account balance is not sufficient, your credit card will be charged the outstanding amount.
```
Comment (easeev) [inline docs/platform/top-up-your-balance-with-crypto.md:13]: charged for?

Comment (akegaviar) [reply docs/platform/top-up-your-balance-with-crypto.md]: fixed

**Review by easeev — APPROVED** [review]
LGTM

**Pull request merged**